web-console/5-components/order-from-catalog.md

```diff
@@ -54,51 +46,58 @@ There are 3 possible steps in this process: Plan, Configuration and Results.
 
 Post 3.6, an additional Bind Step will be implemented.
 
-### Step 1 - Plan
+### Information Step
 
-### User selects a plan
+![template](img/information-step.png)
 
-#### Catalog item has a single version
-![template](img/plan-step.png)
+1. Catalog Item Icon
+	- metadata.imageURL from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Catalog Item Name
+	- metadata.displayName from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Vendor (Optional)
+	- metadata.providerDisplayName from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. List of tags (Optional)
+1. Short description (Optional)
+1. Long description (Optional)
+	- metadata.longDescription from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Image dependencies : Includes an **Image Dependencies** title following by a list of dependencies.  Each image name is preceded by an image icon (note the PF icon)
+1. **View documentation** link which will open a new browser/tab linking to the Documentation URL (Optional)
+	- metadata.documentationUrl from ![OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
 
-#### Catalog item has a multiple versions
+### Plan Step (Optional)
+This step should be shown if there are 1+ plans.
+
+#### Single Plan
 ![template](img/plan-step.png)
 
-#### Implementation Details
-- This step is optional
-- Right column of the content panel includes:
-	- If there are > 1 plans, this step should be added
-	- If multiple versions exist, include version selection on this panel
-		- Version is associated with Builder Images not catalog items
-		- The first version should be selected by default
-	- Each plan should be presented as a radio button option, by the plan title.  
-		- If the plan has a description, it should be listed under the title
-		- The first plan should be selected by default.
+For each plan, display:
+- Plan title
+	- metadata.displayName from ![Plan Metadata Fields](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#plan-metadata-fields)
+- Optional description
+- Features
+	- if the plan has 1+ features, show a **Features** title following by a list of features
+	- metadata.bullets from ![Plan Metadata Fields](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#plan-metadata-fields)
+- Pricing
```

Comment:
Looks like these bullets correspond to the callouts in the image that follows. Maybe use an ordered list so that connection is clear, and maybe the image that follows should come before the explanation.